### PR TITLE
Adding pagecloud functionality to library

### PR DESCRIPTION
**Pull request by jeffreysnell**
This library has been modified to support 8 cardinal direction NSEW crop handles. 

## TODO: 
- [X]  Add styles for NSEW handle support
- [X] Separate logic from driver for existing handle support
- [X] Add new logic to support 8 cardinal directional handles
- [X] Border of viewport should not be part of crop calculations for drag.
- [X] Add flag to disabled croppie's support for cross origin.

## Testing
see https://github.com/pagecloud/pagecloud/pull/6212

### CODE REVIEW

File: croppie.css [hunk 1]
```diff
@@ -72,17 +72,70 @@
 }
 
 .croppie-container .cr-resizer-horisontal {
```
Comment (jimsproull) [inline croppie.css:74]: i'm having trouble getting past this spelling of horizontal :)

File: croppie.js [hunk 2]
```diff
@@ -458,142 +460,356 @@
...
+            if (deltaY > deltaX * originalRatio) {
+                deltaX = inverse * deltaY / originalRatio;
+            } else {
+                deltaY = inverse * deltaX * originalRatio;
+            }
+            return { x: Math.round(deltaX), y: Math.round(deltaY) };
+        }
+
+        function cornerMoveHandler(deltaX, deltaY, inverse, handler1, handler2){
+            const delta = getProportionalDelta(deltaX, deltaY, inverse);
```
Comment (jimsproull) [inline croppie.js:749]: like it

File: croppie.js [hunk 3]
```diff
@@ -1059,7 +1276,28 @@
...
+
+    }
+
+    /**
+     * Get the elements client rect without border width because that causes zoom to be wrong
+     * @param element to get the client rect for
+     */
+    function getInnerDimensions(element) {
+        var copy = getBounds(element);
+        var computedStyles = window.getComputedStyle(element);
```
Comment (jimsproull) [inline croppie.js:1294]: probably won't affect our usecases but if element has a transform that affects width/height (like 'scale') the getComputedStyle won't reflect this. Probably not an issue?

**Review by jimsproull — APPROVED** [review]
seems 🆒 ❄️

File: croppie.js (hunk 3 above)
Comment (jeffreysnell) [reply croppie.js:1294]: Ya, the library just straight up doesn't work on those things so 🍌

**Pull request merged**